splunk_connect_for_snmp/snmp/manager.py

```diff
@@ -475,6 +473,14 @@ def process_snmp_data(self, varBindTable, metrics, target, mapping={}):
                             id.replace('"', ""),
                             mapping.get(f"{mib}::{metric}", mapping.get(mib)),
                         )
+                        if not profile:
```

Comment:
can you just add a comment here about why it is needed?

Reply:
added